scripts/plot_outputs.R

```diff
@@ -9,11 +9,6 @@ ggplot2::theme_set(ggplot2::theme_classic())
 ## Read person properties reports
 ## ===============================#
 
-demographic_report <- readr::read_csv(file.path(
-  "output",
-  "person_demographics_count.csv"
-))
-
 infectious_report <- readr::read_csv(file.path(
   "output",
   "person_infectious_count.csv"
```

Comment:
The report here is called the `person_infectious_count` whereas in `main` it's called `person_property_count`. Please update the report name here.

Reply:
I'm very curious of how hat even happened. We plotted something and "person_infectious_count.csv" does not exist on my computer.